pkg/kubelet/cm/topologymanager/scope.go

```diff
@@ -19,7 +19,7 @@ package topologymanager
 import (
 	"sync"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
remove this alias

Reply:
Seems some files in this package are using alias, and some others don't.Should we unify them?

Reply:
my rule of thumb is to leave imports how it were, unless you are importing something new.